pkg/performanceprofile/controller/performanceprofile_controller_hypershift.go

```diff
@@ -206,6 +206,21 @@ func (r *PerformanceProfileReconciler) hypershiftReconcile(ctx context.Context,
 		return reconcile.Result{}, nil
 	}
 
+	kubeletconfigEncoded, err := encodeKubeletConfig(convertKubeletConfig(componentSet.KubeletConfig))
+	if err != nil {
+		klog.Warningf("failed to Encode KubeletConfig in ConfigMap %s: %v", instance.ObjectMeta.Name, err)
+		// Return and don't requeue
```

Comment:
usual comment about bubbling up and set degraded conditions (seems again a good idea to do so)

Reply:
error part done.
for degraded conditions see https://github.com/openshift/cluster-node-tuning-operator/pull/484#discussion_r1409195931